deps/wazuh_testing/wazuh_testing/__init__.py

```diff
@@ -72,6 +71,9 @@
 T_30 = 30
 T_60 = 60
 
+# Local Internal Options values
+MODULESD_DEBUG = 'wazuh_modules.debug'
+VERBOSE_DEBUG_OUTPUT = '2'
```

Comment:
This variable is already defined. In addition, move `MODULESD_DEBUG` with the other local internal options.

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/3566/commits/1e2362cc96bf3f47e491744e7698a9e63c57ab65